src/main/java/spark/http/matching/MatcherFilter.java

```diff
@@ -92,6 +92,10 @@ public void doFilter(ServletRequest servletRequest,
 
         String httpMethodStr = method.toLowerCase();
         String uri = httpRequest.getPathInfo();
+        if (uri.endsWith("/")){
```

Comment:
can you add a test ?

And what happens if url = "hello/world?color=red&separator=/"